repo_create/test-1.py

```diff
@@ -0,0 +1,96 @@
+#!/usr/bin/python -tt
```

Comment:
I am not sure what is target Python versoin of whole project but we should use Python 3